doc/classes/Control.xml

```diff
@@ -652,24 +754,58 @@
 				Forwarding can be implemented in the target control similar to the methods [method get_drag_data], [method can_drop_data], and [method drop_data] but with two differences:
 				1. The function name must be suffixed with [b]_fw[/b]
 				2. The function must take an extra argument that is the control doing the forwarding
-				[codeblock]
+				[codeblocks]
+				[gdscript]
 				# ThisControl.gd
 				extends Control
+
 				func _ready():
 				    set_drag_forwarding(target_control)
 
 				# TargetControl.gd
 				extends Control
+
 				func can_drop_data_fw(position, data, from_control):
 				    return true
 
 				func drop_data_fw(position, data, from_control):
-				    my_handle_data(data)
+				    my_handle_data(data) # Your handler method.
 
 				func get_drag_data_fw(position, from_control):
 				    set_drag_preview(my_preview)
 				    return my_data()
-				[/codeblock]
+				[/gdscript]
+				[csharp]
+				// ThisControl.cs
+				public class ThisControl : Control
+				{
+				    public override void _Ready()
+				    {
+				        SetDragForwarding(_TargetControl);
```

Comment:
If this is supposed to be public, this should be `targetControl` (or maybe a property `TargetControl`, if this is supposed to be private, this should be `_targetControl` and the GDScript code should be renamed to `_target_control`. Note that neither language example actually declares this variable, it would be nice if you did this.

Reply:
I added it as exported property, but I am not sure about it.

Reply:
You still are referencing `_TargetControl` instead of `TargetControl` on this line.

Reply:
Sorry. Fixed now and rebased